src/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks Condition="'$(DotNetBuildSourceOnly)' != 'true'">netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildSourceOnly)' != 'true'">netstandard2.0;$(NetCoreAppMinTargetFramework)</TargetFrameworks>
     <TargetFramework Condition="'$(DotNetBuildSourceOnly)' == 'true'">netstandard2.0</TargetFramework>
```

IOW, who will be affected by:

```suggestion
    <TargetFramework>$(NetCoreAppMinTargetFramework)</TargetFramework>
```

---

We are - and sourcebuild would. A lot of what lives in this repo is sadly stuck either in NS or has to be dual target. It also has to target the lowest netcoreapp version we have in support. And SB doesn't like that last bit - hence this.